README.md

```diff
@@ -59,13 +59,15 @@ The following props are accepted by the picker:
 | tenorApiKey | `string` | **Reqired** | Tenor v2 API key, obtained from [Google Cloud Console](https://console.cloud.google.com) |
 | onGifClick | `function` | | Callback function that is called when an gif is clicked. The function receives the [`TenorImage`](#tenorimage) object as a parameter. |
 | autoFocusSearch | `boolean` | `true` | Controls the auto focus of the search input. |
-| contentFilter | `ContentFilter` | `ContentFilter.OFF` | Controls the Tenor [Content filtering](https://developers.google.com/tenor/guides/content-filtering) options. If you are using Typescript you can use `ContentFilter` enum. Possible values are `high`, `medium`, `low`, `off`  |
+| contentFilter | `ContentFilter` | `ContentFilter.OFF` | Controls the Tenor [Content filtering](https://developers.google.com/tenor/guides/
+| mediaFilter | `MediaFilter` | `MediaFilter.BASIC` | Reduce the Number of GIF formats returned in the GIF_OBJECT list. If you are using Typescript you can use `MediaFilter` enum. Possible values are `basic`, `minimal`. For image optimizations it's recommended the picker uses the tinygif or nanogif formats to avoid downloading full gif content to the client while searching. On gif select will return the hires `gif`. Read more [Best practices](https://tenor.com/gifapi/documentation#endpoints-search)  |
```

Comment:
There is no point in adding a media filter here if it will still be reduced only to use `tinnygif` in `TenorManager`. This option does nothing?

Reply:
Yea I can remove it from the options and just use basic in the configuration